.github/workflows/maven-verify.yml

```diff
@@ -182,13 +182,10 @@ jobs:
         run:
           mvn --errors --batch-mode --show-version org.apache.maven.plugins:maven-wrapper-plugin:3.1.0:wrapper "-Dmaven=${{ inputs.ff-maven }}"
 
-      - name: Build with Maven
-        run: ./mvnw --errors --batch-mode --show-version ${{ inputs.maven-args }} ${{ inputs.ff-goal }}
-
```

Comment:
should be not removed

Reply:
sorry grrhhh %#%@%^#@*^%# gh web editor (pardon my french)